docs/api/resources/shopping-lists.md

```diff
@@ -123,61 +125,6 @@ For a user with multiple lists:
 ]
 ```
 
-## GET /shopping_lists/:id
```

Comment:
We weren't using this route on the front end at all and don't really have plans to, so I decided to let this one go.